components/cam/tools/mkatmsrffile/Makefile

```diff
@@ -1,11 +1,33 @@
-include $(CASEROOT)/Macros.bluefire
+#-------------------------------------
+# Makefile for mkatmsrffile tool 
+#-------------------------------------
 
-LIBDIR=/ptmp/jedwards/SMS_D.f09_g16.B1850CN.bluefire.C.083815/lib
-LIBS = -L$(LIBDIR) -lcsm_share -lpio -lmct -lmpeu -L$(LIB_NETCDF) -lnetcdf -L$(LIB_PNETCDF) -lpnetcdf
-MODDIR=$(LIBDIR)/include
+#-------------------------------------
+#Instructions:
+# Load a compiler and the associated 
+# netcdf library. Update variables "FC"
+# and "NETCDF_PATH" to add the compiler
+# and path to netcdf home directory
+#-------------------------------------
 
+FC := ifort
+NETCDF_PATH := $(NETCDF_HOME)
 
-mkatmsrffile: mkatmsrffile.F90 
-	$(FC) $(FFLAGS) -I$(MODDIR) $< -o $@  $(LIBS) $(LDFLAGS) 
+EXE := exe_mkatmsrffile
 
+#---------------------------
+# USER INPUT ENDS HERE
+#---------------------------
+
+#include string
+INC = -I$(INC_NETCDF)
+LIB = -L$(LIB_NETCDF) -lnetcdf -lnetcdff
+
+FFLAGS = -O2
```

Comment:
Similar to above, maybe make this `FFLAGS += -02` to allow passing some additional flags via environment variable? Like if we wanted to add bounds checking, without editing the Makefile?